content/en/docs/Concepts/crds/subscription.md

```diff
@@ -39,6 +39,10 @@ spec:
   source: my-catalog
   sourceNamespace: operators
   installPlanApproval: Manual
+  config:
+    env:
+    - name: ARGS
+      value: "-v=10"
```

Comment:
I think my preference would be to have the config-related examples be isolated to their own set of examples, vs. baking it into an existing one. Any thoughts on that approach? I feels like we need to eventually translate https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/subscription-config.md into the olm-docs repository.